app/Helpers/database/set-claim.php

```diff
@@ -441,23 +444,27 @@ function getActiveClaimCount(?string $user = null, bool $countCollaboration = tr
  */
 function updateClaim(int $claimID, int $claimType, int $setType, int $status, int $special, string $claimDate, string $finishedDate): bool
 {
-    sanitize_sql_inputs($claimDate, $finishedDate);
+    $claim = AchievementSetClaim::firstWhere('ID', $claimID);
+    if (!$claim) {
+        return false;
+    }
 
-    $query = "
-        UPDATE
-            SetClaim
-        SET
-            ClaimType = '$claimType',
-            SetType = '$setType',
-            Status = '$status',
-            Special = '$special',
-            Created = '$claimDate',
-            Finished = '$finishedDate',
-            Updated = NOW()
-        WHERE
-            ID = '$claimID'";
+    $oldFinishedDate = $claim->Finished;
+
+    $claim->ClaimType = $claimType;
+    $claim->SetType = $setType;
+    $claim->Status = $status;
+    $claim->Special = $special;
+    $claim->Created = Carbon::parse($claimDate);
+    $claim->Finished = Carbon::parse($finishedDate);
+    $claim->save();
+
+    if ($claim->Finished != $oldFinishedDate) {
```

Comment:
minor: prefer type-safe comparison (`!==`) or add a comment why it isn't.
future increase of PHPStan level will complain about _all_ comparisons that are not type-safe